.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### Ticket
```

Comment:
Issue instead of ticket maybe. Not a strong opinion, it just may be more compatible with wording we use in general. You can decide